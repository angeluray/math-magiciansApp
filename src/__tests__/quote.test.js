import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../pages/quote';

describe('Check tha Quote function renders content', () => {
  it('Check if render the Quote', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
