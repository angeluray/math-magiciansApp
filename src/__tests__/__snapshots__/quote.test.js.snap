// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check tha Quote function renders content Check if render the Quote 1`] = `
<p
  className="quote"
>
  Pure mathematics is, in its way, the poetry of logical ideas.
  <br />
  â€” Albert Einstein, German theoretical physicist
</p>
`;
